onnx/helper.py

```diff
@@ -237,6 +237,9 @@ def make_attribute(
             # Turn np.int32/64 into Python built-in int.
             attr.ints.extend(int(v) for v in value)
             attr.type = AttributeProto.INTS
+        elif all(isinstance(v, numbers.Real) for v in value):
```

Comment:
This supersedes the check at line 233 ```instances(v, float)```, so better to directly replace that with your new check instead of adding a new elif branch.

Reply:
```
if all(isinstance(v, numbers.Integral) for v in value):
...
   INTS
...
elif all(isinstance(v, numbers.Real) for v in value):
  # NB: ints are all instances of numbers.Real,
  # this allows passing mixed of ints and floats values as repeated argument
...
   FLOATS
...
```

Reply:
Good idea. Done.

Reply:
Is it possible to add some comments on the documentation of the function to tell users that when there is such a mix of type, type float is chosen?

Reply:
It's certainly possible. I'm in the middle of moving across the country right now though, so I won't be doing it.